### PR TITLE
Whole rewrite

### PR DESCRIPTION
**Pull request by korewaChino**
Andaman's BuildKit backend is janky and complicated, @lleyton suggested we simply rewrite the whole project to be a simple local CI tool that we can use on off-the-shelf CI providers such GHA and Drone

### CODE REVIEW

File: anda/src/cmd/mod.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (windowsboy111) [inline anda/src/cmd/mod.rs:1]: does this file need to exist or sth

File: anda/src/flatpak.rs [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            FlatpakArtifact::Ref(ref r) => write!(f, "ref {}", r),
+            FlatpakArtifact::Bundle(ref b) => write!(f, "bundle {}", b.display()),
+        }
+    }
+}
+
+pub struct FlatpakBuilder {
+    /// The output directory for the flatpak build
```
Comment (windowsboy111) [inline anda/src/flatpak.rs:25]: why `///` instead of `//`?

File: anda/src/oci.rs [hunk 3]
```diff
@@ -0,0 +1,94 @@
+//! OCI Builder backend
+//! Supports Docker and Podman
+
+use anyhow::{Result};
```
Comment (windowsboy111) [inline anda/src/oci.rs:4]: a

**Review by lleyton — APPROVED** [review]
Ready for mainline

**Pull request merged**